### PR TITLE
Add a maintenance view for running test sql queries on all DBs

### LINKED ISSUE #273: Add maintenance page for running SQL script on all DB's
It is useful to have this feature, to check if there is some corrupted data or if some bug occurs.

### PR DESCRIPTION
**Pull request by mladjo2505**
Closes #273

### CODE REVIEW

File: modules/migration/view/test_sql.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  * {
+    font-size: 12px;
+  }
+  table {
+    border-collapse: collapse;
+  }
+  table, th, td {
+    border: 1px solid black;
+  }
+Try it Yourself »
```
Comment (mladjo2505) [inline modules/migration/view/test_sql.php]: remove

**Comment by mladjo2505** [comment]
Enable only select queries to be run on this!

File: modules/migration/view/test_sql.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+includelogic("migration/migration");
+global $_lib;
+
+$migration_system = new migration_system();
+
+$db_name_restriction = (isset($_REQUEST['db_name'])) ? $_REQUEST['db_name'] : NULL;
+$sql_query = (isset($_REQUEST['sql_query'])) ? $_REQUEST['sql_query'] : NULL;
+$query_empty = !$sql_query;
+
+require_once "record.inc";
```
Comment (mladjo2505) [inline modules/migration/view/test_sql.php]: remove

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

File: modules/migration/model/migration.class.php [hunk 3]
```diff
@@ -72,6 +72,31 @@ function get_databases($with_migrations = true) {
     return $all_migrations;
   }
 
+  function run_script_on_all_db($script) {
+    $model_invoicerecurring_recurring = new model_invoicerecurring_recurring();
```
Comment (mladjo2505) [inline modules/migration/model/migration.class.php:75]: Move the db_list to a separate function

File: modules/migration/view/test_sql.php [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  header('Location: ' . $_lib['sess']->dispatchs . "t=lodo.main");
+  die();
+}
+
+includelogic("migration/migration");
+global $_lib;
+
+$migration_system = new migration_system();
+
+$not_allowed_query_keywords = array('ALTER', 'CREATE', 'DROP', 'TRUNCATE', 'DELETE', 'INSERT', 'UPDATE');
```
Comment (mladjo2505) [inline modules/migration/view/test_sql.php]: Add REPLACE, RENAME, CALL, GRANT, REVOKE, SET, INSTALL, UNINSTALL, KILL, RESET, SHUTDOWN, USE.

**Pull request merged**